includes/wc-payment-api/models/class-wc-payments-api-intention.php

```diff
@@ -40,6 +40,13 @@ class WC_Payments_API_Intention {
 	 */
 	private $status;
 
+	/**
+	 * The ID of charge associated with intention
+	 *
+	 * @var string
+	 */
+	private $charge_id;
```

Comment:
This doesn't really have anything to do with my changes, but I noticed this private variable was missing...and I couldn't really see any good reason why it shouldn't be there. The code still worked, I guess. But anyways, I thought better to add it.